action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
     required: false
     description: 'A comma separated list of languages to exclude'
     default: ''
+  IS_AUTOPUSH:
+    required: false
+    description: 'Whether auto push generated files to desired branch'
+    default: false
```

Comment:
Considering current behavior, the default value should be `true`.

And maybe naming the variable as `AUTO_PUSH` would be better. 🤔 

So it would be like this one below.
```
  AUTO_PUSH:
    required: false
    description: 'Whether auto push generated files to the desired branch'
    default: true
   ```

Reply:
Yes, I think the default true value is better so that users won't confuse.